.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,19 @@
+name: .NET Core
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.2.108
```

Comment:
please include multiple dot-net versions that we want to support.

Reply:
I removed the 3 dotnet core version. We can investigate adding .net core 3 and windows framework when we pick up this ticket. For now it is similar to travis.